packages/supersearch/src/lib/components/SuperSearch.svelte

```diff
@@ -63,6 +63,16 @@
 		}
 	});
 
+	const searchStatus = StateEffect.define<{ message: string }>({});
+
+	$effect(() => {
+		if (search.data) {
+			const effects = { effects: searchStatus.of({ message: 'new_data' }) };
```

Comment:
I'm thinking we maybe could clarify what `new_data` is? E.g. `new_response_data` or `new_mapping_data` or something else? But `new_data` is also fine...

Reply:
Agreed. Also a bit shaky with the arbitrary string. In https://github.com/libris/lxlviewer/pull/1186/commits/39d57bbc63a892e6cac0a308516366e7ab4bb0dc I export a constant for messages, so we don't fail because of spelling errors in one place.

Reply:
Double effects should be fixed in https://github.com/libris/lxlviewer/pull/1186/commits/8f8cb5a83126155e92d9786e5dc6543b190a2905